docs/partial_source/contributing/setting_up.rst

```diff
@@ -390,11 +395,32 @@ Now, if Hypothesis detects an error in the code it will return more detailed inf
 For questions, please reach out on the `setting up discussion`_
 or on `discord`_ in the `docker channel`_!
 
+Setting up for Free
+-------------------
+
+PyCharm professional is only free for students, a good alternative would be using Visual Studie Code if you don't have a student account. The most easiest and the most efficient way would be using Visual Studio Code with the Docker extension. You'll hopefully be done with this in no time.
```

Comment:
Could you please frame the first sentence as `Visual Studio Code is a recommended free alternative to setting up, especially if you're not eligible for a student license with PyCharm Professional`?

Reply:
Done!